lnwire/shutdown.go

```diff
@@ -71,11 +80,8 @@ func (s *Shutdown) MaxPayloadLength(pver uint32) uint32 {
 	// Len - 2 bytes
 	length += 2
 
-	// ScriptPubKey - 34 bytes for pay to witness script hash
-	length += 34
-
-	// NOTE: pay to pubkey hash is 25 bytes, pay to script hash is 22
-	// bytes, and pay to witness pubkey hash is 22 bytes in length.
+	// ScriptPubKey
+	length += deliveryAddressMaxSize
```

Comment:
missing length prefix?

Reply:
Isn't the +2 on [line 81](https://github.com/lightningnetwork/lnd/pull/3655/files#diff-b928d202d95ccbba631a8cd4fe621844R81) the length prefix?